cms/templates/cms/program_page.html

```diff
@@ -5,8 +5,248 @@
+                                                         id="faqs-{{ forloop.counter }}"
+                                                         aria-labelledby="exampleHeadingContinuousOne"
+                                                         role="tabpanel">
+                                                        <div class="panel-body">
+                                                            {{ faq.answer }}
+                                                        </div>
+                                                    </div>
+                                                </div>
+                                            {% endfor %}
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                        <div role="tabpanel" class="tab-pane fade in" id="courses">
+                            <div class="panel-body">
+                                <div class="row">
+                                    <div class="col-md-10">
+                                        <h3>Courses</h3>
+                                        <div class="panel-group panel-group-continuous" id="exampleAccordionContinuous"
+                                             aria-multiselectable="true" role="tablist">
+                                            {% for course in page.Courses.all %}
+                                                <div class="panel">
+                                                    <div class="panel-heading" id="exampleHeadingContinuousOne"
+                                                         role="tab">
+                                                        <a class="panel-title" data-parent="#exampleAccordionContinuous"
+                                                           data-toggle="collapse"
+                                                           href="#course-{{ forloop.counter }}"
+                                                           aria-controls="exampleCollapseContinuousOne"
+                                                           aria-expanded="true">
+                                                            {{ course.title }}
+                                                        </a>
+                                                    </div>
+                                                    <div class="panel-collapse collapse in"
+                                                         id="course-{{ forloop.counter }}"
+                                                         aria-labelledby="exampleHeadingContinuousOne"
+                                                         role="tabpanel">
+                                                        <div class="panel-body">
+                                                            {{ course.description }}
```

Comment:
It looks like this is outputting escaped HTML. Can you figure out how this should work so we see the formatted output here?